code/game/machinery/doors/windowdoor.dm

```diff
@@ -334,6 +334,21 @@
 		if("touch")
 			INVOKE_ASYNC(src, .proc/open_and_close)
 
+/obj/machinery/door/window/rcd_vals(mob/user, obj/item/construction/rcd/the_rcd)
+	switch(the_rcd.mode)
+		if(RCD_DECONSTRUCT)
+			return list("mode" = RCD_DECONSTRUCT, "delay" = 50, "cost" = 32)
+	return FALSE

```

Comment:
Why are you doing this instead of just one if?

Reply:
It’s just how every other RCD vals list was made for every other object even ones with only one application.